content/ja/docs/concepts/configuration/secret.md

```diff
@@ -11,411 +11,311 @@ weight: 30
+このSecret型を使用する場合は、Secretの`data`フィールドに`.dockercfg`キーが含まれていることを確認する必要があります。このキーの値は、base64形式でエンコードされた`~/.dockercfg`ファイルの内容です。
 
-```
-MWYyZDFlMmU2N2Rm
-```
+`kubernetes.io/dockerconfigjson`型は、`~/.dockercfg`の新しいフォーマットである`~/.docker/config.json`ファイルと同じフォーマットルールに従うシリアル化されたJSONを保存するために設計されています。
+このSecret型を使用する場合、Secretオブジェクトの`data`フィールドには`.dockerconfigjson`キーが含まれている必要があります。このキーでは、`~/.docker/config.json`ファイルのコンテンツがbase64でエンコードされた文字列として提供されます。
 
-このようなSecretを書きます。
+以下は、`kubernetes.io/dockercfg`型のSecretの例です。
 
 ```yaml
 apiVersion: v1
 kind: Secret
-metadata:
-  name: mysecret
-type: Opaque
-data:
-  username: YWRtaW4=
-  password: MWYyZDFlMmU2N2Rm
+  name: secret-dockercfg
+type: kubernetes.io/dockercfg
+ data:
+  .dockercfg: |
+    "<base64 encoded ~/.dockercfg file>"
 ```
 
-これでSecretを[`kubectl apply`](/docs/reference/generated/kubectl/kubectl-commands#apply)コマンドで作成できるようになりました。
+{{< note >}}
+base64エンコーディングを実行したくない場合は、代わりに`stringData`フィールドを使用することを選択できます。
+{{< /note >}}
+
+マニフェストを使用してこれらの型のSecretを作成すると、APIserverは期待されるキーが`data`フィールドに存在するかどうかを確認し、提供された値を有効なJSONとして解析できるかどうかを確認します。APIサーバーは、JSONが実際にDocker configファイルであるかどうかを検証しません。
+
+
+Docker configファイルがない場合、または`kubectl`を使用してDockerレジストリSecretを作成する場合は、次の操作を実行できます。
 
 ```shell
-kubectl apply -f ./secret.yaml
+kubectl create secret docker-registry secret-tiger-docker \
+  --docker-username=tiger \
+  --docker-password=pass113 \
+  --docker-email=tiger@acme.com
 ```
 
-出力は次のようになります。
+このコマンドは、`kubernetes.io/dockerconfigjson`型のSecretを作成します。
+`data`フィールドから`.dockerconfigjson`コンテンツをダンプすると、その場で作成された有効なDocker configである次のJSONコンテンツを取得します。
 
-```
-secret "mysecret" created
+```json
+{
+  "auths": {
+    "https://index.docker.io/v1/": {
+      "username": "tiger",
+      "password": "pass113",
+      "email": "tiger@acme.com",
+      "auth": "dGlnZXI6cGFzczExMw=="
+    }
+  }
+}
 ```
 
-状況によっては、代わりに`stringData`フィールドを使いたいときもあるでしょう。
-このフィールドを使えばBase64でエンコードされていない文字列を直接Secretに書くことができて、その文字列はSecretが作られたり更新されたりするときにエンコードされます。
+### Basic authentication Secret
 
-実用的な例として、設定ファイルの格納にSecretを使うアプリケーションをデプロイすることを考えます。
-デプロイプロセスの途中で、この設定ファイルの一部のデータを投入したいとしましょう。
+`kubernetes.io/basic-auth`型は、Basic認証に必要な認証を保存するために提供されています。このSecret型を使用する場合、Secretの`data`フィールドには次の2つのキーが含まれている必要があります。
 
-例えば、アプリケーションは次のような設定ファイルを使用するとします。
+- `username`: 認証のためのユーザー名
+- `password`: 認証のためのパスワードかトークン
 
-```yaml
-apiUrl: "https://my.api.com/api/v1"
-username: "user"
-password: "password"
-```
+上記の2つのキーの両方の値は、base64でエンコードされた文字列です。もちろん、Secretの作成に`stringData`を使用してクリアテキストコンテンツを提供することもできます。
+
+次のYAMLは、Basic authentication Secretの設定例です。
 
-次のような定義を使用して、この設定ファイルをSecretに保存することができます。
 
 ```yaml
 apiVersion: v1
 kind: Secret
 metadata:
-  name: mysecret
-type: Opaque
+  name: secret-basic-auth
+type: kubernetes.io/basic-auth
 stringData:
-  config.yaml: |-
-    apiUrl: "https://my.api.com/api/v1"
-    username: {{username}}
-    password: {{password}}
+  username: admin
+  password: t0p-Secret  config.yaml: |-
```

Comment:
[In original](https://v1-19.docs.kubernetes.io/docs/concepts/configuration/secret/#basic-authentication-secret), there is not `config.yaml: |-`.
Is it overlook? :thinking: 
```suggestion
  password: t0p-Secret
```